Rules.md

```diff
@@ -144,36 +145,36 @@ player available, all write down their action and execute that action.
 
 ## Moving units
 
-Units can move to an adjacent region. If this adjacent region has at least one coin invested and
-if it was already friendly prior to resolving the move token, the unit may move through this
-region to another region. Units cannot end in friendly regions owned by other players.
+Units can move to an adjacent region. If this adjacent region has at least one invested coin and
+was already friendly prior to resolving the move token, the unit may move through it to a next region
+adjacent to this invested region. Units cannot end in friendly regions owned by other players.
 
 *Examples:*
 
-* *My infantry in England can move through N-France (occupied by an ally) to Benelux because both
-  England and N-France have an invested coin.*
-* *If I have 3 units in the Middle East and there are coins invested in unoccupied Egypt and the
-  Middle East, I cannot go from the Middle East to Libya with a single move token. This is because
-  Egypt is not a friendly region before resolving this move token.*
+* *My infantry in England can move through N-France (occupied by an ally) to Benelux because
+  N-France has an invested coin.*
+* *If I have 3 units in the Middle East and there are coins invested in unoccupied Egypt, I cannot
+  go from the Middle East to Libya with a single move token. This is because Egypt is not a friendly
+  region before resolving this move token.*
 
 ### Plundering
 
 If a unit's move ends in a previously unoccupied region, invested coins may be (partly) stolen.
 
 ### Sea regions
 
-Units of different teams can reside in the same sea region simultaneously.
+Units of different players and even teams can reside in the same sea region simultaneously.
 
-### Ambiguous crossings
+### Straits
 
-The following points on the map can be crossed both by land-land and sea-sea movement:
+The following land-land borders can also be crossed by sea-sea movement, as indicated on the map:
 
-* **Denmark islands**
-* **English Channel**
-* **Gibraltar**
-* **Istanbul**
+* **Sweden - Denmark** (the Sound)
+* **England - North France** (Strait of Dover)
```

Comment:
English Channel / Strait of Dover / Pas de Calais? :p

I would be fine with 'Strait of Dover / Pas de Calais', similar to the flavor we added for the empires.